src/pages/content-edit/validator.js

```diff
@@ -0,0 +1,137 @@
+import m from "mithril";
+import toArray from "lodash.toarray";
+
+import debounce from "lodash.debounce";
+
+import css from "./validator.css";
+
+function allInputs(form) {
```

Comment:
```js
return toArray(document.querySelectorAll("input, select, textarea"));
```

And since it's only used once this entire function can be removed.